src/java/picard/analysis/directed/HsMetrics.java

```diff
@@ -73,58 +73,58 @@
     /** The number of PF reads that are not marked as duplicates. */
     public long PF_UNIQUE_READS;
 
-    /** PF reads / total reads.  The percent of reads passing filter. */
+    /** The fraction of reads passing the vendor's filter (PF reads/total reads).   */
     public double PCT_PF_READS;
 
-    /** PF Unique Reads / Total Reads. */
+    /** The fraction of unique reads from the total number of PF-reads [PF (Unique reads/Total reads)]. */
     public double PCT_PF_UQ_READS;
 
-    /** The number of PF unique reads that are aligned with mapping score > 0 to the reference genome. */
+    /** The number of PF-unique reads that aligned to the reference genome with a mapping score > 0. */
     public long PF_UQ_READS_ALIGNED;
 
-    /** PF Reads Aligned / PF Reads. */
+    /** The fraction of PF reads aligned to the reference genome from the total number of PF reads. */
     public double PCT_PF_UQ_READS_ALIGNED;
 
-    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
+    /** The number of PF unique bases that are aligned to the reference genome with mapping scores > 0. */
```

Comment:
Hi Yossi:

Thanks for reviewing these.  Please do not review the Illumina-specific tools, those tools are part of a different PR.

Thanks and have a great weekend!

> On May 27, 2016, at 8:55 PM, Yossi Farjoun <notifications@github.com> wrote:
> 
> In src/java/picard/analysis/directed/HsMetrics.java <https://github.com/broadinstitute/picard/pull/549#discussion_r64981043>:
> 
> >      public double PCT_PF_UQ_READS_ALIGNED;
> >  
> > -    /** The number of PF unique bases that are aligned with mapping score > 0 to the reference genome. */
> > +    /** The number of PF unique bases that are aligned to the reference genome with mapping scores > 0. */
> score, not scores.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/broadinstitute/picard/pull/549/files/a4c48fd5af9c8472ea9d4826cc6a0c6175598e61#r64981043>, or mute the thread <https://github.com/notifications/unsubscribe/AH4LeNRjXTqeRsCH2YnvCT4sQIVZK2jHks5qF5KegaJpZM4IowZ8>.
> 

--
David E. Kling, Ph.D.
Scientific/Technical Writer
Broad Institute
75 Ames Street, #7003X
Cambridge, Ma  02142
O (617) 714-7351
C (617) 291-1617